docs/source/getting_started.rst

```diff
@@ -36,6 +36,85 @@ appropiate agent with the environmet related to your problem domain and the lib
         state, reward, done, info = env.step(action)
 
 
+Saving experiments
+------------------
+
+``RLib`` :ref:`class <RLib Class>` provides an API for saving your experiment in a compressed ``.lz4`` format.
+You can later reconstruct the experiment state and continue from the exact point where you have ended or you can
+alter some training parameters during the reloading process.
+
+
+Full reconstruction
+~~~~~~~~~~~~~~~~~~~
+
+We can imagine a scenario, where we set up the experiment, perform a little training, and then we need to make a break,
+so we save the experiment at some state that we would later want to carry on from. When we are ready to continue with
+the training, we can load the whole experiment to a new RLib instance.
+
+.. code-block:: python
+
+    import reinforced_lib as rfl
+
+    from reinforced_lib.agents import ThompsonSampling
+    from reinforced_lib.exts import IEEE_802_11_ax
+    
+    # Setting up the experiment
+    rl = rfl.RLib(
+        agent_type=ThompsonSampling,
+        ext_type=IEEE_802_11_ax
+    )
+
+    # Do some training
+    # ...
+
+    # Saving experiment state for later
+    rl.save("<checkpoint-path>")
+
+    # Do some other staff, quit the script if you want.
+
+    # Load the saved training
+    rl = RLib.load(self.checkpoint_path)
```

Comment:
To taka drobnostka, ale jeśli to ma być w założeniu fragment jakiegoś skryptu, to tam nie będzie `self`. Można w to miejsce wrzucić tak jak wyżej `"<checkpoint-path>"`.

Reply:
racja